scripts/test/Muon/threading_manager_test.py

```diff
@@ -0,0 +1,303 @@
+from __future__ import (absolute_import, division, print_function)
+
+import sys
+import time
+
+import unittest
+if sys.version_info.major == 3:
+    from unittest import mock
+else:
+    import mock
+
+from Muon.GUI.Common.threading_manager import *
```

Comment:
Can we avoid `*` imports - they are bad practice so it would be good to avoid new ones.